Weird, errors were all in restore, so might be something else:

```
C:\dotnetbuild\work\A8520933\w\9E7608B5\e\RunTests\RunTests.csproj : error NU1102: Unable to find package Microsoft.WindowsDesktop.App.Ref with version (= 5.0.0-preview.5.20253.1)
C:\dotnetbuild\work\A8520933\w\9E7608B5\e\RunTests\RunTests.csproj : error NU1102:   - Found 17 version(s) in nuget.org [ Nearest version: 5.0.0-preview.4.20251.1 ]
  Failed to restore C:\dotnetbuild\work\A8520933\w\9E7608B5\e\RunTests\RunTests.csproj (in 463 ms).
```

--- a/.azure/pipelines/helix-matrix.yml
+++ b/.azure/pipelines/helix-matrix.yml
@@ -4,13 +4,14 @@ trigger:
   branches:
     include:
     - master
+    - release/*
 
 # Run PR validation for now
 pr:
   autoCancel: true
   branches:
     include:
-    - helix/winarm
+    - '*'
     
 variables:
 - ${{ if ne(variables['System.TeamProject'], 'internal') }}:

--- a/.azure/pipelines/helix-matrix.yml
+++ b/.azure/pipelines/helix-matrix.yml
@@ -44,25 +44,25 @@ jobs:
       path: artifacts/log/
       publishOnError: true
       
-  # Helix ARM64
-  - template: jobs/default-build.yml
-    parameters:
-      jobName: Helix_arm64_daily
-      jobDisplayName: "Tests: Helix ARM64 Daily"
-      agentOs: Linux
-      timeoutInMinutes: 180
-      steps:
-      # Build the shared framework
-      - script: ./restore.sh -ci -nobl
-        displayName: Restore
-      - script: ./build.sh -ci --nobl --arch arm64 -test --no-build-nodejs --all -projects $(Build.SourcesDirectory)/eng/helix/helix.proj /p:IsHelixJob=true /p:IsHelixDaily=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
-        displayName: Run build.sh helix arm64 target
-        env:
-          HelixApiAccessToken: $(HelixApiAccessToken) # Needed for internal queues
-          SYSTEM_ACCESSTOKEN: $(System.AccessToken) # We need to set this env var to publish helix results to Azure Dev Ops
-      installNodeJs: false
-      artifacts:
-      - name: Helix_arm64_logs
-        path: artifacts/log/
-        publishOnError: true
-        includeForks: true      
+# Helix ARM64
+- template: jobs/default-build.yml
+  parameters:
+    jobName: Helix_arm64_daily
+    jobDisplayName: "Tests: Helix ARM64 Daily"
+    agentOs: Linux
+    timeoutInMinutes: 180
+    steps:
+    # Build the shared framework
+    - script: ./restore.sh -ci -nobl
+      displayName: Restore
+    - script: ./build.sh -ci --nobl --arch arm64 -test --no-build-nodejs --all -projects $(Build.SourcesDirectory)/eng/helix/helix.proj /p:IsHelixJob=true /p:IsHelixDaily=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+      displayName: Run build.sh helix arm64 target
+      env:
+        HelixApiAccessToken: $(HelixApiAccessToken) # Needed for internal queues
+        SYSTEM_ACCESSTOKEN: $(System.AccessToken) # We need to set this env var to publish helix results to Azure Dev Ops
+    installNodeJs: false
+    artifacts:
+    - name: Helix_arm64_logs
+      path: artifacts/log/
+      publishOnError: true
+      includeForks: true

--- a/.azure/pipelines/helix-matrix.yml
+++ b/.azure/pipelines/helix-matrix.yml
@@ -5,6 +5,13 @@ trigger:
     include:
     - master
 
+# Run PR validation for now
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - helix/winarm
+    
 variables:
 - ${{ if ne(variables['System.TeamProject'], 'internal') }}:
   - name: _UseHelixOpenQueues

